projects/almalinux-8/almalinux-8-mariadb-10.11/docker-compose.yml

```diff
@@ -18,6 +18,11 @@ services:
       - catalog
     volumes:
       - shared_volume:/irods_testing_environment_mount_dir
+    # The core test suite for iRODS uses SIGHUP to reload the server's configuration often.
+    # To keep the iRODS Rule Language from failing and producing RE_UNABLE_TO_READ_SESSION_VAR,
+    # we bump the maximum amount of shared memory from 64mb to 100mb. This is required because
+    # iRODS 5 can have two Agent Factories running simultaneously (due to the SIGHUP).
+    shm_size: 100mb
```

Comment:
Any reason for 100 over... say... 128?

Reply:
No reason other it is enough space for the reload to succeed. I think iRODS uses ~40MB for shared memory. The bump to 100 felt decent. And there's a little wiggle room in there for additional shared memory if needed.

128 is just as good IMO. I don't have strong feeling around this.

Reply:
Okay. No strong feeling here - was just thinking the double is cleaner.

Reply:
I like powers of 2 as well :-)

If I had to guess, `shm_size` is probably a limit check rather than an instruction to docker to allocate _N_ bytes.

Reply:
Ok. Going to leave at 100 for now. We can always change it later if needed.